packages/module/src/module.ts

```diff
@@ -29,49 +27,31 @@ export default defineNuxtModule<ModuleOptions>({
   async setup({ prefix, componentDir }, nuxt) {
     const COMPONENT_DIR_PATH = componentDir!
     const ROOT_DIR_PATH = nuxt.options.rootDir
-    const { resolve, resolvePath } = createResolver(ROOT_DIR_PATH)
+    const UTILS_ALIAS = '@/lib/utils' // Use the same path from the cli for backward compatibility
+    const { resolve } = createResolver(ROOT_DIR_PATH)
+    const logger = useLogger('shadcn-nuxt')
 
+    // `lib/utils`
+    const utilsTemplate = addTemplate({
+      filename: 'shadcn-nuxt/utils.ts',
+      getContents: () => UTILS,
+      write: true,
+    })
+    nuxt.options.alias = { [UTILS_ALIAS]: utilsTemplate.dst, ...nuxt.options.alias } // We add our alias to the top to avoid conflicts with nuxt built in `@` alias
+
+    // Recommend to remove the root `lib/utils` if exists
+    const isRootUtilsExists = await findPath('./lib/utils.ts', { cwd: ROOT_DIR_PATH })
+    if (isRootUtilsExists)
+      logger.warn('[shadcn-nuxt] `lib/utils.ts` is auto generated by the module and can be safely removed.')
+
+    // Components Auto Imports
+    const componentsPath = resolve(COMPONENT_DIR_PATH)
     nuxt.hook('components:dirs', (dirs) => {
       dirs.unshift({
-        path: resolve(COMPONENT_DIR_PATH),
-        extensions: [],
+        path: componentsPath,
+        extensions: ['.vue'], // Scan `.vue` only and skip `.ts` files as we use them inside components file only
```

Comment:
With this, some of the components that exported directly from `index.ts` will be ignored in auto-import process like shadcn-vue form

Reply:
Didn't test forms 👀 I'll revert it until a better solution comes up